src/pages/Projects/allProjectsGQL.js

```diff
@@ -133,4 +133,13 @@ export const currenciesMarketSegmentsGQL = gql`
   }
 `
 
+export const ALL_ERC20_PROJECTS_QUERY = gql`
```

Comment:
@GarageInc Do we have this in our queries/ directory?
Does this query unique only for this place?

Reply:
This query used in BalancesView component(checking wallet balance) and in TriggerForm